src/components/reservation/ReservationItem.js

```diff
@@ -0,0 +1,44 @@
+import PropTypes from 'prop-types';
+import { SwiperSlide } from 'swiper/react';
+
+const ReservationItem = ({ reservation }) => (
+  <SwiperSlide>
+    <div className="d-flex reserve">
+      <div className="w-100">
+        <div className="res-detail">
+          <div className="d-flex spc-btw">
+            <p>Model:</p>
+            <p>{reservation.car.name}</p>
+          </div>
+          <div className="d-flex spc-btw">
+            <p>City:</p>
+            <p>{reservation.city}</p>
+          </div>
+          <div className="d-flex spc-btw">
+            <p>Date:</p>
+            <p>{reservation.date}</p>
+          </div>
+        </div>
+      </div>
+      <div className="">
+        <div className="d-res-image w-100">
+          <img src={reservation.car.image} className="r-image" alt="img" />
+        </div>
+      </div>
+      <div className="d-flex w-100 h-center">
+        <button
+          type="button"
+          className="btn-cancel"
+        >
+          Cancel
```

Comment:
- [ ] OPTIONAL: I suggest that since you have a button, implement the cancel functionality for reservations.

Otherwise, I suggest just removing the button in order to prevent confusion.  This is optional. 👍